### DISCUSSION TITLE
Unraid Docker keeps stopping

### DISCUSSION THREAD

**Discussion by RonnieBlaze** [Taxel/PlexTraktSync#754] [Support] [answered]
Is it normal for the docker to keep stopping its self after the script run?

**Comment by glensc** [comment, marked as answer]
We can't read your mind or have access to your system. what kind of support do you expect if you don't bother explaining what exactly did you do what docs you followed, etc?

**Reply by thedinz** [reply]
This is going to be the same issue I came here for, he needs to utilize the watch argument
